forms.html

```diff
@@ -2,10 +2,11 @@
 <html>
 	<head>
 		<title>
-			Introduction to Forms
+			Introduction to Forms in PHP
 		</title>
 	</head>
-	<body>
+	<body>\

```

Comment:
remove the slash at the end of body tag.

Reply:
thanks for informing me